contracts/plugins/assets/RTokenAsset.sol

```diff
@@ -43,6 +43,7 @@ contract RTokenAsset is IAsset {
     /// @return high {UoA/tok} The high price estimate
     function tryPrice() external view virtual returns (uint192 low, uint192 high) {
         (uint192 lowBUPrice, uint192 highBUPrice) = basketHandler.price(); // {UoA/BU}
+        assert(low <= high); // not obviously true just by inspection
```

Comment:
Did you mean to check lowBUprice and highBUPrice?

Reply:
Good catch!